dask/dataframe/groupby.py

```diff
@@ -166,7 +166,11 @@ def _groupby_slice_apply(df, grouper, key, func, *args, **kwargs):
     # No need to use raise if unaligned here - this is only called after
     # shuffling, which makes everything aligned already
     group_keys = kwargs.pop("group_keys", True)
-    g = df.groupby(grouper, group_keys=group_keys)
+    dropna = kwargs.pop("dropna", None)
```

Comment:
Now that we have dropped support for Python 2 we can probably include these keywords explicitly in the signature.  Previously we avoided this because they were after a `*args`, which was not valid Python syntax.

Reply:
Sounds good - I moved the keywords into the signature (in most cases).